### PR TITLE
Generate int64 type for int64 instead of string

### PR DESCRIPTION
**Pull request by jenngeorge**
Generate int64 type for int64 instead of string

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -8,7 +8,8 @@ generate-openapi-client:
 			-g go \
 			-i internal/spec/openapi.json \
 			-o internal/openapi-generator \
-			-c internal/spec/config.yaml
+			-c internal/spec/config.yaml \
+			--type-mappings=int64=int64
```
Comment (jenngeorge) [inline Makefile:12]: I'd like this to be in https://github.com/cockroachdb/cockroach-cloud-sdk-go/blob/main/internal/spec/config.yaml instead if possible.

**Review by erademacher — APPROVED** [review]
Nice!!

File: Makefile (hunk 1 above)
Comment (jenngeorge) [reply Makefile]: Updated.

**Comment by jenngeorge** [comment]
TFTR!

**Pull request merged**